api/agentpb/agent.proto

```diff
@@ -11,6 +11,13 @@ import "inventorypb/agent_status.proto";
 import "inventorypb/agents.proto";
 import "inventorypb/services.proto";
 
+// TextFiles contains files which can be used to connect to DB (certificates, keys and etc).
+message TextFiles {
+  map<string, string> files = 1;
+  string template_left_delim = 2;
+  string template_right_delim = 3;
```

Comment:
What is this? What is template and for what this delimiters required for?

Can we some comments here?

Reply:
template_left_delim and template_right_delim are used to replace placeholders in templates(DSN and etc.) with paths to files.

Reply:
IMHO better to add some comments with explanation of purpose of these fields.

Reply:
Done